example/example.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,293 @@
+// !$*UTF8*$!
```

Comment:
I we should go with the pattern of having two commits here:

* One to remove the old README, which is simply https://github.com/ably/tutorials/commit/597562b0cf3b366a18bdca924b67ce369e41a030
* Then we need this commit to be titled "Step 2a) Set up Xcode project" which needs to be referenced in the tutorial at https://github.com/ably/docs/pull/168/commits/a1167cad479560c8cbc045faee444eb1c95822a1. This is an important step for users in stepping through the tutorial and is unfortunately missed.

The tutorials are intended to be a step-by-step guide for people to get up and running with a project.  Each commit is a reference to the step so that they can check it out and do it themselves, but they also need to be able to just do the entire set up themselves so it's important we walk them through from start to finish explaining each step.